feature-group-definitions/js-modules.dist.yml

```diff
@@ -0,0 +1,34 @@
+# Generated from: js-modules.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: JavaScript modules
+description: 'JavaScript modules allow code to be organized into reusable units. Modules use `import` to load other modules and `export` to declare what is available to import from other modules. In HTML, modules are loaded with `<script type="module">`.'
+caniuse: es6-module
+spec:
+  - https://html.spec.whatwg.org/multipage/webappapis.html#javascript-module-script
+  - https://tc39.es/ecma262/multipage/ecmascript-language-scripts-and-modules.html#sec-modules
+snapshot: ecmascript-2015
+usage_stats:
+  - https://chromestatus.com/metrics/feature/timeline/popularity/2062
+  - https://chromestatus.com/metrics/feature/timeline/popularity/2615
+status:
+  baseline: high
+  baseline_low_date: 2018-05-09
+  baseline_high_date: 2020-11-09
+  support:
+    chrome: "61"
+    chrome_android: "61"
+    edge: "16"
+    firefox: "60"
+    firefox_android: "60"
+    safari: "11"
```

Comment:
Safari was 10.1 before.

Reply:
It's because of `nomodule` and I think it's actually correct to treat this as an essential part of the feature and bump the Safari version slightly here. AIUI, this attribute is the fallback mechanism and important to deploying JS modules in practice. I'll add TODO about this though.